MUXSDKKaltura/MUXSDKKalturaTests/MUXSDKStatsTests+Utilities.swift

```diff
@@ -0,0 +1,96 @@
+//
+//  MUXSDKStatsTests+Utilities.swift
+//  MUXSDKKalturaTests
+//
+//  Created by Audra Rodriguez on 18/11/21.
+//  Copyright © 2021 Mux, Inc. All rights reserved.
+//
+
+import XCTest
+@testable import MUXSDKKaltura
+import MuxCore
+
+extension MUXSDKStatsTests {
+    private enum EventType{
+        case customerVideoData
+        case customerPlayerData
+        case customerData
+        case customData
+    }
+    
+    func assertDispatchedEventTypesForPlayer(id: String, expectedEventTypes: [String]) {
+        let dispatchedEventsType = MockedDispatcher.shared.dispatchedEvents
+            .filter { $0.playerId == id }
+            .map { $0.event.getType() }
+        
+        XCTAssertEqual(expectedEventTypes, dispatchedEventsType)
+    }
+    
+    func assertDispatchedCustomerDataEventsAtIndex(
+        index: Int,
+        expectedCustomerVideoData: [String : Any]? = nil,
+        expectedCustomerPlayerData: [String : Any]? = nil,
+        expectedCustomerViewData: [String : Any]? = nil,
+        expectedCustomData: [String : Any]? = nil
+    )  {
+        let dispatchedEvent = MockedData.dispatcher.dispatchedEvents[index].event as? MUXSDKDataEvent
+        if let expectedCustomerVideoData = expectedCustomerVideoData {
```

Comment:
there is a lot of repeated code here, perhaps we can do the nil check and XCTAssertNil in assertCustomerData?